web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -112,8 +118,20 @@ export const SidebarCardsList = (props: Props): ReactElement => {
 
   return (
     <>
-      <h2 className="h1-styling margin-top-0">{Config.dashboardDefaults.sidebarHeading}</h2>
-      <hr className="margin-top-2 margin-bottom-3 bg-cool-lighter" aria-hidden={true} />
+      {isDesktopAndUp ? (
+        <>
+          <h2 className="margin-top-0 text-normal-override">
```

Comment:
[pebble/dust] Instead of doing an `h2` with a text override, what if we did `<div role="heading" aria-level={2}` like we do in other places in the app? That way, it's still a level 2 header for accessibility purposes, but we don't need to override the h2 styling. What do we think?